uaclient/cli/config.py

```diff
@@ -52,11 +54,11 @@ def action_config_show(args, *, cfg, **kwargs):
     row_tmpl = "{key: <" + col_width + "} {value}"
 
     for key in config.UA_CONFIGURABLE_KEYS:
-        print(row_tmpl.format(key=key, value=getattr(cfg, key, None)))
+        print(row_tmpl.format(key=key, value=getattr(pro_config, key, None)))
 
-    if (cfg.global_apt_http_proxy or cfg.global_apt_https_proxy) and (
-        cfg.ua_apt_http_proxy or cfg.ua_apt_https_proxy
-    ):
+    if (
+        pro_config.global_apt_http_proxy or pro_config.global_apt_https_proxy
```

Comment:
Even if you use my `to_dict` suggestion above, still do this direct attribute reference for this if statement.
The cases are:
1. if key is a variable, use the dict
2. if the key we care about is known at authoring time, use the attribute on the object

Reply:
^